### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by bdaylik**
### What does it do?

Added Turkish translation. No code changes.

### Why is it needed?

To better serve Turkish speaking users.

### How to test it?

Just select Turkish language in your profile settings.

### Related issue(s)/PR(s)

None I know of.

### CODE REVIEW

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
hi @bdaylik - that's a huge and great addition, thank you!
left a small comment about a translation file not being translated 😊

File: packages/plugins/i18n/admin/src/translations/tr.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "CMEditViewCopyLocale.copy-failure": "Failed to copy locale",
```
[the remark this answers is not in the document]
Comment (bdaylik) [reply packages/plugins/i18n/admin/src/translations/tr.json:2]: oops missed that one, good catch.
I'll get back to work then :)

[the remark this answers is not in the document]
Comment (bdaylik) [reply packages/plugins/i18n/admin/src/translations/tr.json]: I've translated the I18N plugin as well. Can you recheck?

**Comment by bdaylik** [comment]
@ronronscelestes Looks like all checks are passed. Can you please merge this in?

**Comment by SalahAdDin** [comment]
Thanks big bro, it helps a lot!

**Pull request merged**